src/codecatalyst/activation.ts

```diff
@@ -96,3 +98,39 @@ export async function activate(ctx: ExtContext): Promise<void> {
         authProvider.removeSavedConnection()
     })
 }
+
+async function showReadmeFileOnFirstLoad(
+    workspaceState: vscode.ExtensionContext['workspaceState'],
+    projectName: CodeCatalystProject['name']
+): Promise<void> {
+    // Check dev env state to see if this is the first time the user has connected to a dev env
+    const isFirstLoad = workspaceState.get('aws.codecatalyst.devEnv.isFirstLoad', true)
+
+    if (!isFirstLoad) {
+        return
+    }
+
+    // Determine expected readme file location
+    let readmePath = `README.md`
+    if (isCloud9()) {
+        readmePath = projectName + '/' + readmePath
```

Comment:
I suppose this should be reliable. Else we could tell findFiles() to recursive into child folders.

Reply:
should we use path.join() here?